cargo build --no-default-features fails

The crate fails to build with default features disabled (as required for use in `no_std` crates):

```bash
rust-radio-sx127x> cargo build --no-default-features
    Updating crates.io index
   Compiling proc-macro2 v1.0.8
   Compiling unicode-xid v0.2.0
   Compiling syn v1.0.14
   Compiling log v0.4.8
   Compiling cfg-if v0.1.10
   Compiling nb v0.1.2
   Compiling void v1.0.2
   Compiling bitflags v1.2.1
   Compiling serde v1.0.104
   Compiling libc v0.2.66
   Compiling embedded-hal v0.2.3
   Compiling embedded-spi v0.5.8
   Compiling quote v1.0.2
   Compiling async-trait v0.1.24
   Compiling serde_derive v1.0.104
   Compiling radio v0.4.4
   Compiling radio-sx127x v0.10.0 (/home/zac/Repos/rust-radio-sx127x)
error[E0599]: no method named `round` found for type `f32` in the current scope
   --> src/fsk.rs:148:64
    |
148 |         let fdev = ((channel.fdev as f32) / device::FREQ_STEP).round() as u32;
    |                                                                ^^^^^ method not found in `f32`

error[E0599]: no method named `round` found for type `f32` in the current scope
   --> src/fsk.rs:149:75
    |
149 |         let datarate = (self.config.xtal_freq as f32 / channel.br as f32).round() as u32;
    |                                                                           ^^^^^ method not found in `f32`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0599`.
error: could not compile `radio-sx127x`.

To learn more, run the command again with --verbose.
```
Last working tag appears to be `0.7.4`, where plain integer truncation was used for fdev and datarate, e.g.

```rust
let fdev = ((channel.fdev as f32) / device::FREQ_STEP) as u32;
```
If rounding is absolutely required it can be provided by the `libm` crate with some compile-time configuration.

--- a/src/fsk.rs
+++ b/src/fsk.rs
@@ -144,9 +144,21 @@ where
         // Set frequency
         self.set_frequency(channel.freq)?;
 
+        // Need our own rounding alogrithm because corelib doesn't have
+        // f32::round
+        let round = |x: f32| -> u32 {
+            let integer = x as u32;
+                if (x - (integer as f32)) < 0.5 {
+                   integer
+                }
+                else {
+                   integer + 1
+                }
+        };
+
         // Calculate channel configuration
-        let fdev = ((channel.fdev as f32) / device::FREQ_STEP).round() as u32;
-        let datarate = (self.config.xtal_freq as f32 / channel.br as f32).round() as u32;
+        let fdev = round((channel.fdev as f32) / device::FREQ_STEP);
+        let datarate = round(self.config.xtal_freq as f32 / channel.br as f32);
         trace!("fdev: {} bitrate: {}", fdev, datarate);
 
         // Set frequency deviation

